src/Bicep.Core.Samples/Files/InvalidDisableNextLineDiagnosticsDirective_CRLF/main.diagnostics.bicep

```diff
@@ -2,8 +2,40 @@
+
 #  disable-next-line  no-unused-params
 //@[0:1) [BCP001 (Error)] The following token is not recognized: "#". (CodeDescription: none) |#|
 //@[0:1) [BCP007 (Error)] This declaration type is not recognized. Specify a parameter, variable, resource, or output declaration. (CodeDescription: none) |#|
 param storageAccount2 string = 'testStorageAccount'
 //@[6:21) [no-unused-params (Warning)] Parameter "storageAccount2" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-params)) |storageAccount2|
+
+/* comment before */#disable-next-line no-unused-params
+param storageAccount3 string = 'testStorageAccount'
+
+#disable-next-line/* comment between */ no-unused-params
+//@[18:18) [BCP226 (Error)] Expected at least one diagnostic code at this location. Valid format is "#disable-next-line diagnosticCode1 diagnosticCode2 ..." (CodeDescription: none) ||
+//@[39:40) [BCP001 (Error)] The following token is not recognized: " ". (CodeDescription: none) | |
+//@[39:40) [BCP007 (Error)] This declaration type is not recognized. Specify a parameter, variable, resource, or output declaration. (CodeDescription: none) | |
+param storageAccount4 string = 'testStorageAccount'
+//@[6:21) [no-unused-params (Warning)] Parameter "storageAccount4" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-params)) |storageAccount4|
+
+//#disable-next-line no-unused-params
+param storageAccount5 string = 'testStorageAccount'
+//@[6:21) [no-unused-params (Warning)] Parameter "storageAccount5" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-params)) |storageAccount5|
+
+#disable-next-line 
+//@[19:19) [BCP226 (Error)] Expected at least one diagnostic code at this location. Valid format is "#disable-next-line diagnosticCode1 diagnosticCode2 ..." (CodeDescription: none) ||
+no-unused-params
+//@[0:2) [BCP007 (Error)] This declaration type is not recognized. Specify a parameter, variable, resource, or output declaration. (CodeDescription: none) |no|
+param storageAccount6 string = 'testStorageAccount'
+//@[6:21) [no-unused-params (Warning)] Parameter "storageAccount6" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-params)) |storageAccount6|
+
+#madeup-directive
+//@[0:1) [BCP001 (Error)] The following token is not recognized: "#". (CodeDescription: none) |#|
+//@[0:1) [BCP007 (Error)] This declaration type is not recognized. Specify a parameter, variable, resource, or output declaration. (CodeDescription: none) |#|
+param storageAccount7 string = 'testStorageAccount'
+//@[6:21) [no-unused-params (Warning)] Parameter "storageAccount7" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-params)) |storageAccount7|
+
+var terminatedWithDirective = 'foo' #disable-next-line no-unused-params

```

Comment:
Is it intentional here that we allow this directive in-line after existing statements?

Reply:
Good question. Other than readability I don't see any issues. Do you think we should not allow this?

Reply:
It does look pretty weird and Bicep is very opinionated about newlines currently, so we should block it in my opinion.

Reply:
(Especially if it's being ignored also.)

Reply:
Hmmm it does suppress the warning on next line. But I could look into blocking it.

Reply:
Added check to block it. Let me know what you think.
https://github.com/Azure/bicep/commit/92218a732b3a5b601fe369d05df48328d9462076